src/include/rados.h

```diff
@@ -517,6 +517,7 @@ struct ceph_osd_op {
 			__le64 ver;     /* no longer used */
 			__u8 op;	/* CEPH_OSD_WATCH_OP_* */
 			__u32 gen;      /* registration generation */
+			__u32 timeout; /* connection timeout */
```

Comment:
@tchaikov, you are right. There is compile error after adding the static check. Do I need to find another place to pass timeout to OSD?

Following is the compile error.

In file included from /home/ryne/ceph/src/include/ceph_fs.h:16:0,
                 from /home/ryne/ceph/src/include/types.h:32,
                 from /home/ryne/ceph/src/msg/msg_types.h:20,
                 from /home/ryne/ceph/src/common/entity_name.h:24,
                 from /home/ryne/ceph/src/common/config.h:24,
                 from /home/ryne/ceph/src/common/dout.h:20,
                 from /home/ryne/ceph/src/compressor/AsyncCompressor.cc:15:
/home/ryne/ceph/src/include/rados.h:573:1: error: static assertion failed: sizeof(ceph_osd_op) breaks the compatibility
 static_assert(sizeof(ceph_osd_op) == (2+4+(8+8+8)+4),